src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.spec.ts

```diff
@@ -337,7 +339,14 @@ describe('DotEditLayoutDesignerComponent', () => {
 
         describe('should show', () => {
             beforeEach(() => {
-                component.pageState = new DotRenderedPageState(mockUser, mockDotRenderedPage);
+                component.pageState = new DotRenderedPageState(mockUser, {
```

Comment:
@hmoreras the problem wasn't the order, it was that you need to set the ```anonymous``` to false in order to get the dialog shown.